catalogue_pipeline/id_minter/src/main/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDao.scala

```diff
@@ -24,23 +24,22 @@ class IdentifiersDao @Inject()(db: DB, identifiers: IdentifiersTable)
    * source identifiers.
    */
   def lookupId(
-    sourceIdentifier: SourceIdentifier,
-    ontologyType: String
+    sourceIdentifier: SourceIdentifier
```

Comment:
Just from a reviewing standpoint, it would be easier if the ID minter changes were in a separate PR. :-)

Reply:
Not really possible. The id minter changes where needed because we introduced the `Identified` `Identifiable` and `Unidentifiable` wrapper classes. If we didn't do this in the same PR the id minter would crap out for every work with creators or publishers with identifiers

Reply:
I've just had the same conversation with Ricardo – model/ID minter changes are fine together, would be nice if we could get them separate from the MARC-parsing creators stuff, but understand that might be tricky.

Reply:
Yeah, I think that's a bit tricky at this point